webcurator-submit-to-rosetta/src/main/java/nz/govt/natlib/ndha/wctdpsdepositor/preprocessor/ArcIndexProcessor.java

```diff
@@ -45,7 +45,7 @@
 public class ArcIndexProcessor implements PreDepositProcessor {
     private static final Log log = LogFactory.getLog(ArcIndexProcessor.class);
 
-    private static final boolean APPEND_TO_FILE = true;
+    private static final boolean APPEND_TO_FILE = false;
```

Comment:
What issue does this solve? (I'm having trouble seeing the bigger picture)

Reply:
This fixed an issue about depositing collections to Rosetta. When depositing to Rosetta, the CDX will be recreated and joined together. Some CDX was missing and failed to deposit to Rosetta.